x-pack/plugins/security_solution/public/common/components/exceptions/__examples__/index.stories.tsx

```diff
@@ -27,14 +32,12 @@ storiesOf('components/exceptions', module)
     ];
 
     return (
-      <ThemeProvider theme={() => ({ eui: euiLightVars, darkMode: false })}>
-        <ExceptionItem
-          commentsAccordionId={'accordion--comments'}
-          exceptionItem={payload}
-          handleDelete={() => {}}
-          handleEdit={() => {}}
-        />
-      </ThemeProvider>
+      <ExceptionItem
+        commentsAccordionId={'accordion--comments'}
+        exceptionItem={payload}
+        handleDelete={() => {}}
```

Comment:
nit: My preference is to suffix these incoming callback props with `on`, and then suffix a component's internal callbacks with `handle`:

```tsx
// consumer 
<MyModal onClose={whateverConsumerWants} />

// in MyModal
const handleClose = useCallback(() => {
  handleInternalState();
  onClose();
}, [handleInternalState, onClose]);
return <EuiModal onClose={handleClose} />;
```

It's an arbitrary rule, but I find that it helps me with consistency and in distinguishing between internal/external functions.

Reply:
I like that pattern. I went ahead and updated them to follow this.